dev/devicelab/bin/tasks/build_android_host_app_with_module_aar.dart

```diff
@@ -267,7 +264,7 @@ class ModuleTest {
           <String>['app:assembleDebug'],
           environment: <String, String>{
             'JAVA_HOME': javaHome,
-            'FLUTTER_ANALYTICS_LOG_FILE': analyticsOutputFile.path,
+            'FLUTTER_SUPPRESS_ANALYTICS': 'true',
```

Comment:
I doubt `FLUTTER_SUPPRESS_ANALYTICS` is actually needed since we have bot detection elsewhere, but I would rather delete all of these in a separate PR.